pkg/secretlock/env/env_secret_lock.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package env
+
+import (
+	"crypto/cipher"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/google/tink/go/subtle/random"
+	"golang.org/x/crypto/chacha20poly1305"
+
+	"github.com/hyperledger/aries-framework-go/pkg/secretlock"
+)
+
+// Lock is a secret lock service responsible for encrypting keys using a master key stored as environment variable
+type Lock struct {
+	aead cipher.AEAD
+}
+
+// NewService creates a new instance of a local secret lock service
+func NewService(keyURI string) (secretlock.Service, error) {
+	mk := os.Getenv("LOCAL_" + strings.ReplaceAll(keyURI, "/", "_"))
```

Comment:
the prefix should be an argument - normally representing the end application name.

Reply:
also i do not understand why keyURI is passed in here.

shouldn’t the env key be constructed in the other functions based on keyURI?

Reply:
sure, I will make it a prefix argument

Reply:
done